_bibliography/pint.bib

```diff
@@ -2060,6 +2060,25 @@ @article{ArteagaEtAl2015
 	year = {2015}
 }
 
+@article{Bedez2015,
+title = "A fully parallel in time and space algorithm for simulating the electrical activity of a neural tissue ",
+journal = "Journal of Neuroscience Methods ",
+volume = "",
+number = "",
+pages = " - ",
+year = "2015",
+note = "",
+issn = "0165-0270",
+doi = "http://dx.doi.org/10.1016/j.jneumeth.2015.09.017",
+author = "Mathieu Bedez and Zakaria Belhachmi and Olivier Haeberl\'e and Renaud Greget and Saliha Moussaoui and Jean-Marie Bouteiller and Serge Bischoff",
+keywords = "GPU",
+keywords = "Neural activity",
+keywords = "Parareal algorithm",
+keywords = "MPI",
+keywords = "Partial differential equations "
+}
+
```

Comment:
Could you please change to `{}` instead of `""` for consistency and include the abstract?